tests/docker_cluster.py

```diff
@@ -18,26 +18,39 @@
 and not directly call into the docker-py API.
 """
 
+import errno
+import os
+import shutil
 import subprocess
 import sys
-import shutil
-import os
-import errno
 import uuid
 from time import sleep
 
 from docker import Client
 from docker.errors import APIError
 from docker.utils.utils import kwargs_from_env
+from retrying import retry
 
 from prestoadmin import main_dir
 from tests.base_cluster import BaseCluster
 from tests.product.constants import \
-    DEFAULT_DOCKER_MOUNT_POINT, DEFAULT_LOCAL_MOUNT_POINT, \
-    BASE_TD_IMAGE_NAME
+    DEFAULT_DOCKER_MOUNT_POINT, DEFAULT_LOCAL_MOUNT_POINT
 
 DIST_DIR = os.path.join(main_dir, 'tmp/installer')
 
+_DOCKER_START_TIMEOUT = 30000
+_DOCKER_START_WAIT = 1000
+
+ENSURE_STARTED_OPT_OUT_IMAGES = [
```

Comment:
Could you rename this?  It took me a while to figure out what you meant by opt out. What about NON_CENTOS_IMAGES?  The point is just that we don't check about centos services because it's not a centos container.